cottoncandy/interfaces.py

```diff
@@ -410,7 +416,8 @@ def download_json(self, object_name):
         json_data : dict
             Dictionary representation of JSON file
         """
-        assert self.exists_object(object_name)
+        if not self.exists_object(object_name):
```

Comment:
For all of these, maybe the S3/GDrive interface should return the appropriate error. i.e. `S3ObjectNotFound` or `GDriveObjectNotFound`